PHPCI/Application.php

```diff
@@ -133,7 +133,13 @@ protected function setLayoutVariables(View &$layout)
     {
         /** @var \PHPCI\Store\ProjectStore $projectStore */
         $projectStore = b8\Store\Factory::getStore('Project');
-        $layout->projects = $projectStore->getAll();
+        $layout->projects = $projectStore->getWhere(
+          array('archived' => (int) isset($_GET['archived'])),
```

Comment:
I think it make sense when `$_GET[archive]` has multiple values, it's good for `status` parameter, like: unpublished, published, archived or any other, in this case it's logical.